From 84879cc75a663edb4d8061e7e4cd4e2983e7ca42 Mon Sep 17 00:00:00 2001
From: Hui Xia <hui.xia@intel.com>
Date: Wed, 15 Aug 2018 00:24:18 -0400
Subject: [PATCH 2074/2345] media: lc898122: fix compile issue when PM disabled

The compile issue was found in m68k-allmodconfig build
which disable CONFIG_PM.

Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/media/i2c/lc898122/lc898122.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/drivers/media/i2c/lc898122/lc898122.c b/drivers/media/i2c/lc898122/lc898122.c
index 41cff00..1abf559 100644
--- a/drivers/media/i2c/lc898122/lc898122.c
+++ b/drivers/media/i2c/lc898122/lc898122.c
@@ -172,6 +172,7 @@ int lc898122_read_long(struct i2c_client *client, u16 reg, u32 *val)
 	return 0;
 }
 
+#ifdef CONFIG_PM
 static int lc898122_adjust_af_parameters(struct lc898122_device *lc898122_dev)
 {
 	struct i2c_client *client = lc898122_dev->client;
@@ -247,6 +248,9 @@ static int lc898122_adjust_af_parameters(struct lc898122_device *lc898122_dev)
 
 	return 0;
 }
+#else
+#define lc898122_adjust_af_parameters NULL
+#endif
 
 
 /*
@@ -512,6 +516,7 @@ static int lc898122_remove(struct i2c_client *client)
 	return 0;
 }
 
+#ifdef CONFIG_PM
 static int lc898122_poweron_init(struct lc898122_device *lc898122_dev)
 {
 	int rval = 0;
@@ -536,7 +541,6 @@ static int lc898122_poweron_init(struct lc898122_device *lc898122_dev)
 	return rval;
 }
 
-#ifdef CONFIG_PM
 static void lc898122_complete(struct device *dev)
 {
 	struct i2c_client *client = to_i2c_client(dev);
@@ -598,6 +602,7 @@ static int lc898122_runtime_resume(struct device *dev)
 }
 
 #else
+#define lc898122_poweron_init		NULL
 #define lc898122_complete		NULL
 #define lc898122_runtime_suspend	NULL
 #define lc898122_runtime_resume		NULL
-- 
2.7.4

